activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -54,6 +55,10 @@ def initialize(connection, logger, connection_options, config)
         super(connection, logger, config)
       end
 
+      def init_schema_cache
+        MySQL::SchemaCache.new(self)
```

Comment:
Why are we defining a special schema cache for MySQL? I think we don't need that.